### ISSUE TITLE
Make a library

### ISSUE THREAD

**Issue by Tunas1337** [whoeverest/nlbcli#14]
Having a CLI tool to access my bank account is amazing, and I salute you for thinking to do what many of us have always wanted to, yet none of us have had the courage to do.
However, I can think of many use cases for a Python library that does the same task, for integration in more complex tools without relying on calling "nlbcli" in shell scripts (or, in Python, `os.system`) and using regex and slicing (for example, `echo -e Имаш уште само $(nlbcli accounts balance | grep "Available balance:" | tail -c +20 | head -c -8) денари,\\nсмири се со трошењето` can be done much more elegantly with a Python library, not to mention it can be more functional.)

Please let me know what you think.

**Comment by whoeverest** [comment]
Agreed! This being a library _and_ a CLI has been on my mind since the start, but in true MVP fashion I wanted to release it sooner and receive feedback. :)

Do you have any experience with python packaging / modules / making Python act sane both in CLI mode and as a library? Any advice? If you want to do a PR let me know. I'll eventually do this myself, but help is always appreciated.
